app/src/main/java/jp/co/cyberagent/stf/query/DoCleanBluetoothBondedDevicesResponder.java

```diff
@@ -0,0 +1,89 @@
+package jp.co.cyberagent.stf.query;
+
+import java.lang.reflect.Method;
+import java.util.Set;
+import android.content.Context;
+import android.os.Build;
+import android.bluetooth.BluetoothDevice;
+import android.bluetooth.BluetoothAdapter;
+import android.bluetooth.BluetoothManager;
+import android.util.Log;
+
+import com.google.protobuf.GeneratedMessageLite;
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import jp.co.cyberagent.stf.proto.Wire;
+
+
+public class DoCleanBluetoothBondedDevicesResponder extends AbstractResponder {
+    private static final String TAG = DoCleanBluetoothBondedDevicesResponder.class.getSimpleName();
+
+    public DoCleanBluetoothBondedDevicesResponder(Context context) {
+        super(context);
+    }
+
+    @Override
+    public GeneratedMessageLite respond(Wire.Envelope envelope) throws InvalidProtocolBufferException {
+        Wire.DoCleanBluetoothBondedDevicesRequest request =
+                Wire.DoCleanBluetoothBondedDevicesRequest.parseFrom(envelope.getMessage());
+
+        boolean successful = false;
```

Comment:
IMHO this variable is useless. I would just inline true/false values in `buildResponse` invocations.